distro.py

```diff
@@ -1074,7 +1074,71 @@ def _parse_distro_release_content(line):
         return distro_info
 
 
-_distro = LinuxDistribution()
+def _normalize_id(distro_id, table):
+    """ Helper function which normalizes a distro id value. """
+    distro_id = distro_id.lower().replace(' ', '_')
+    return table.get(distro_id, distro_id)
+
+
+class LinuxMintDistribution(LinuxDistribution):
+    """ LinuxMint will have the same /etc/os-release file
+    as it's upstream Ubuntu so we should not use any info
+    from that file except `ID_LIKE`. """
+    def os_release_attr(self, attribute):
+        if attribute == 'id_like':
+            v = super(LinuxMintDistribution, self).os_release_attr(attribute)
+            return v
+        return ''
+
+
+class CloudLinuxDistribution(LinuxDistribution):
+    def id(self):
```

Comment:
They happen to be constants for `CloudLinuxDistribution` but that's not necessarily the case for all, most distributions have to get the information about `id` from `os_release_attr`, `lsb_release_attr`, and `distro_release_attr` function calls.

I know we can just use the `@property` decorator but there are more "constant" values than just `id`.